README.rst

```diff
@@ -24,6 +24,16 @@ Wheels, bdists, eggs and accompanying PGP-signatures can be uploaded
 either with ``pip``, ``setuptools``, ``twine``, ``pypi-uploader``, or simply copied
 with ``scp``.
 
+The official software powering PyPI_ is Warehouse_. However, Warehouse_
+is fairly specialized to be ``pypi.org``'s own software, and should not
+be used in other contexts. In particular, it does not officially support
+being used as a custom package index by users wishing to serve their own
+packages.
+
+``pypiserver`` implements the same interfaces as `PyPI`_, allowing
+standard Python packaging tooling such as ``pip`` and ``twine`` to
+interact with it as a package index just as they would with PyPI_, while
+making it much easier to get a running index server.
```

Comment:
It might be nice to stick this in a `note` directive so that it's got some visual separation:

```suggestion
.. note::
   The official software powering PyPI_ is Warehouse_. However, Warehouse_
   is fairly specialized to be ``pypi.org``'s own software, and should not
   be used in other contexts. In particular, it does not officially support
   being used as a custom package index by users wishing to serve their own
   packages.

   ``pypiserver`` implements the same interfaces as `PyPI`_, allowing
   standard Python packaging tooling such as ``pip`` and ``twine`` to
   interact with it as a package index just as they would with PyPI_, while
   making it much easier to get a running index server.
```

Reply:
Cool! Done, clicking on the magic button.